src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/InProcOrRemoteHostAnalyzerRunner.cs

```diff
@@ -22,16 +22,13 @@ namespace Microsoft.CodeAnalysis.Diagnostics;
 
 internal sealed class InProcOrRemoteHostAnalyzerRunner
 {
-    private readonly bool _enabled;
     private readonly IAsynchronousOperationListener _asyncOperationListener;
     public DiagnosticAnalyzerInfoCache AnalyzerInfoCache { get; }
 
     public InProcOrRemoteHostAnalyzerRunner(
```

Comment:
>public InProcOrRemoteHostAnalyzerRunner( [](http://example.com/codeflow?start=4&length=40)

shouldn't a ctor call into this have changed?

Reply:
yes?